backend/app/main.py

```diff
@@ -1,9 +1,11 @@
 import sentry_sdk
 from fastapi import FastAPI
 from fastapi.routing import APIRoute
+from sqlmodel import Session
```

Comment:
Why is session in main.py?  Main should  not be hitting the database

Reply:
removed

Reply:
removed